src/bootstrap/bin/rustc.rs

```diff
@@ -194,6 +194,8 @@ fn main() {
                 // do that we pass a weird flag to the compiler to get it to do
                 // so. Note that this is definitely a hack, and we should likely
                 // flesh out rpath support more fully in the future.
+                //
+                // TODO: remove condition after next stage0
```

Comment:
This is unrelated to this PR's primary changes, right? Just want to make sure...